Serilog.Sinks.RichTextBox.WinForms/Sinks/RichTextBoxForms/Extensions/RichTextBoxExtensions.cs

```diff
@@ -70,6 +72,13 @@ public static void AppendRtf(this RichTextBox richTextBox, string rtf, bool auto
             richTextBox.SelectionLength = 2;
             richTextBox.SelectedText = SpaceCharacter;
 
+            if (richTextBox.Lines.Length > maxLogLines)
+            {
+                richTextBox.SelectionStart = 0;
+                richTextBox.SelectionLength = richTextBox.Lines[..^maxLogLines].Select(x => x.Length + 1).Sum();
+                richTextBox.SelectedText = SpaceCharacter;
```

Comment:
Instead of using `SpaceCharacter` it might be better to use `Environment.NewLine`. For some reason it causes some misalignment with the other lines, placing a new line instead is an easy fix.

![image](https://github.com/vonhoff/Serilog.Sinks.RichTextBox.WinForms.Colored/assets/55744345/73d52fa4-cde2-4f9a-8cf4-8e1eb9baf5e5)

Reply:
Good catch! Unfortunately `Environment.NewLine` didn't resolve it, but a null string (character) did: `"\0"`.  Will push an update shortly.